README.md

```diff
@@ -110,6 +110,8 @@ Usage of tusd:
     	Use Google Cloud Storage with this bucket as storage backend (requires the GCS_SERVICE_ACCOUNT_FILE environment variable to be set)
   -hooks-dir string
     	Directory to search for available hooks scripts
+  -hooks-enabled-events string
+    	Comma separated list of enabled hook events, set to "-" to disable all (default "*")
```

Comment:
> set to "-" to disable all

Is that correct? Isn't "" used to disable all hooks?

Reply:
Correct, I used `-` in a previous version and forgot to update it in the docs, I fixed that in this [commit - c93e37f](https://github.com/tus/tusd/pull/302/commits/c93e37f592fd9322924827d6eb79ef9dfe6768a8)  which I added to the pull request later.